src/EditModel/event-program/tracker-program/program-stages/EditProgramStage.js

```diff
@@ -1,35 +1,61 @@
-import React from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import { compose, lifecycle } from 'recompose';
 
-import ProgramStageStepper from './ProgramStageStepper';
+import { getInstance } from 'd2/lib/d2';
```

Comment:
I don't think this import is being used?

Reply:
Good catch